### PR TITLE
Qexp func test 

### PR DESCRIPTION
**Pull request by czwcandy**

### CODE REVIEW

**Comment by sandygross** [comment]
Hi @czwcandy thank you very much! I have a few comments, that we may can implement. Let me know what you think: 
E.g. by plugging in the value 1 I get an error, which I do not really know what it means. Could you pls check that?

File: src/mathdistops/qexp.py [hunk 1]
```diff
@@ -1,4 +1,7 @@
-def qexp(p, rate, plot_graph=False):
```
Comment (sandygross) [inline src/mathdistops/qexp.py]: I would suggest, we rename the argument to `graph` and set it by default to True, such that it is consistent with the other functions. I think it is a good idea to add a default value for rate; e.g. 'rate=1'

Comment (sandygross) [reply src/mathdistops/qexp.py]: What do u think @carinaya, as you created the other function qexp ?

Comment (carinaya) [reply src/mathdistops/qexp.py]: we can do that, and if you do so, please let me know your final function expression. And I can update the readme file afterward

Comment (czwcandy) [reply src/mathdistops/qexp.py]: Hi Sandy, Thank you for your comment. Can you pls try again? I modified qexp fucntion and test function both. On my local, p=1 cases passed, and coverge is 100%

**Comment by sandygross** [comment]
Hi Katherine, thanks for updating the function. Unfortuantely, there is still an issues with the graph…If you plot it it looks weird in the sense that the graph is not on the full x-axis (see picture). This because even more extreme when the rate is different from 1… Could you pls fix that? Maybe it is helpful if you use the PDF and CDF that Kun created for her function qexp.
Image 2024-01-18 at 9 25 AM

**Review by carinaya — APPROVED** [review]

**Pull request merged**